makefile

```diff
@@ -230,6 +235,9 @@ TESTS += $(TESTS:%=%f)
 test: $(TESTS)
 ifneq ($(MINGW),ON)
 # non-Windows: it will fail if either of these return nonzero exit code...
+    ifeq ($(MSYS),ON)
```

Comment:
same issue as above. Something to explain that msys is grouped with linux/osx. The word "non-Windows" is wrong.  thanks.

Reply:
Yes, "non-windows" should be changed.

Reply:
ok.. I await it.. thanks, Alex